meta/schemas/device-type.yaml

```diff
@@ -0,0 +1,19 @@
+$id: https://openwrt.org/schema/device-type.yaml
+$schema: http://json-schema.org/draft-07/schema#
+description: Type of device
+title: device-type
+type: string
+enum:
+  - Camera
+  - Emulator
+  - Modem
+  - NAS
```

Comment:
You don't use `SBC` for Single Board Computer, but you do use `NAS`, maybe this should be fixed for consistency?